### ISSUE TITLE
No provider for VirtualScrollComponent!

### ISSUE THREAD

**Issue by victorbonilla** [rintoj/angular2-virtual-scroll#120]
I am trying to implement this example https://github.com/rintoj/angular2-virtual-scroll#focus-on-an-item but all I get is the following error:

`ERROR Error: Uncaught (in promise): Error: No provider for VirtualScrollComponent!`

I am importing VirtualScrollModule in the component module that it is actually using this component. I tried to importing into higher level modules, but with no luck

**Comment by GlennVanSchil** [comment]
Same issue here

**Comment by GlennVanSchil** [comment]
I managed to get it working in a sub-module of my project. 

1. I've added the package to my project `yarn add angular2-virtual-scroll`
2. Included in my sub-module.module.ts
```
imports: [
        CommonModule,
        VirtualScrollModule
    ]
```
3. Used it in my sub-module's root component sub-module.component.html as described in the read me
4. an additional step I had to do was declare the viewPortList in my typescript as well.

My IDE did keep complaining though, but when I executed yarn run start it was successful. I also did **not** add anything in my app.module.ts
